src/pages/LogView/LoadingPage/index.tsx

```diff
@@ -30,30 +31,63 @@ const LoadingPage: React.FC<LoadingPageProps> = ({ logType }) => {
     [slugs.execution]: execution,
   } = useParams();
   const dispatchToast = useToastContext();
-  const { ingestLines } = useLogContext();
+  const { ingestLines, setLogMetadata } = useLogContext();
```

Comment:
It seems like there's a console error coming from this file. It's possible you might have to set the metadata in the `useEffect` below instead

<img width="686" alt="Screen Shot 2022-10-21 at 4 20 52 PM" src="https://user-images.githubusercontent.com/47064971/197282795-999b3de8-d958-422d-b74a-f5b26c828109.png">

Reply:
Good catch! Fixed